### ISSUE TITLE
[Keyboard Navigation - Home]: The keyboard focus order is incorrect on the home page.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66243]
### User Experience:  
It would be difficult for the Keyboard dependent users who navigate the application sequentially and expect the focus order to be consistent with the sequential reading order. It will make the users confused if the same screen opens twice while activating different controls.

### Repro Steps:
Open #Url: https://play.grafana.org/   Sign in with valid credentials. "Home" page appears. Navigate the page. Navigate the header controls such as "Add Panel" and select it. Navigate the "Add panel" pop-up. Verify the issue.

### Actual Result: The keyboard focus order is incorrect on the home page.
Observation:  On selecting the "Add panel" button, the keyboard focus does not move to the "Add panel" pop-up. It remains on the "Add panel" button.  From the "Add panel" button it takes 7 tabs it moves to the "Add panel" pop-up.

### Expected Result:
The keyboard focus order should be correct on the Home page.
Solution: On selecting the "Add panel" button, the keyboard focus should move to the "Add panel" pop-up.
Note:  Same issue exists with Absolute time range blade. Same issue exits in the "Explore" page i.e After navigating all the controls, focus is moving to expanded tooltip information instead of directly moving to the tool tip after expanding the 'Click to show description' button. Same issue exists under 'Delete' pop up. Same issue exists under Alert rules tab.

### Severity
`SEV2`

**Comment by mgifford** [comment]
Always nice to have the visual for this from Accessibility Insight's TabStop interface. There are other tools too, but this really helps show problems for keyboard only users.

**Comment by natellium** [comment]
The Add panel issue is solved from 9.5 onwards. @polibb Do you think we should create independent issues for the ones in the note?

>  Same issue exists with Absolute time range blade. Same issue exits in the "Explore" page i.e After navigating all the controls, focus is moving to expanded tooltip information instead of directly moving to the tool tip after expanding the 'Click to show description' button. Same issue exists under 'Delete' pop up. Same issue exists under Alert rules tab.
